### PR TITLE
Avoid duplication of Impact stage acceptance criteria

### PR DESCRIPTION
**Pull request by tobie**

### CODE REVIEW

**Review by joesepi — APPROVED** [review]

**Review by dylans — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by kborchers — APPROVED** [review]

File: PROJECT_PROGRESSION.md [hunk 1]
```diff
@@ -206,10 +206,9 @@ Impact Stage projects are expected to participate actively in CPC proceedings, a
 To graduate from At Large or Growth status, or for a new project to join as an Impact project, a project must meet the Growth stage criteria plus:
 
  * Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. 
- * Have a documented and publicly accessible description of the project's governance, decision-making, and release processes.
+ * Have a documented and publicly accessible description of the project's governance, decision-making, contribution, and release processes. This is preferably laid out in a GOVERNANCE.md file and references a CONTRIBUTING.md and OWNERS.md file showing the current and emeritus committers.
```
Comment (sendilkumarn) [inline PROJECT_PROGRESSION.md:209]: nit: It will be awesome to link the documents here.

**Review by sendilkumarn — APPROVED** [review]

File: PROJECT_PROGRESSION.md (hunk 1 above)
Comment (tobie) [reply PROJECT_PROGRESSION.md:209]: We’d need to have template or example documents which we don’t have right now. Maybe file a separate issue about it?

Comment (eemeli) [reply PROJECT_PROGRESSION.md:209]: This sounds like it's asking for the "contribution processes" to be "laid out in a GOVERNANCE.md file", but I presume that the intent here is for that to be in the CONTRIBUTING.md, which is then referred-to from GOVERNANCE.md, yes?

**Pull request merged**